File: coding/28th February 2025/Max Consecutive Ones.java
//Max Consecutive Ones
class Solution 
{
    public int findMaxConsecutiveOnes(int[] nums) 
    {
        int c = 0, max = 0;
        for(int ele : nums)
        {
            if(ele == 1)
            {
                c++;
                if(c > max)
                    max = c;
            }
            else
                c = 0;
        }
        return max;
    }
}

File: Coding in Python/05th January 2026/Max Consecutive Ones.py
#Max Consecutive Ones
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        c = 0
        maxi = 0
        for ele in nums:
            if ele == 1:
                c += 1
            else:
                c = 0
            if c > maxi:
                maxi = c
        return maxi
